pylint/checkers/classes/class_checker.py

```diff
@@ -2026,7 +2026,10 @@ def is_abstract(method: nodes.FunctionDef) -> bool:
             if name in node.locals:
                 # it is redefined as an attribute or with a descriptor
                 continue
-            self.add_message("abstract-method", node=node, args=(name, owner.name))
+
+            self.add_message(
+                "abstract-method", node=node, args=(name, owner.name, node.name)
+            )
```

Comment:
```suggestion
            self.add_message(
                "abstract-method", node=node, args=(name, owner.name, node.name), confidence=INFERENCE
            )
```

You might need to import this.

Reply:
added